pkg/serviceaccount/tokens_controller.go

```diff
@@ -44,18 +44,16 @@ type TokensControllerOptions struct {
 	// SecretResync is the time.Duration at which to fully re-list secrets.
 	// If zero, re-list will be delayed as long as possible
 	SecretResync time.Duration
-}
 
-// DefaultTokenControllerOptions returns
-func DefaultTokenControllerOptions(tokenGenerator TokenGenerator) TokensControllerOptions {
```

Comment:
Having a default option helper function lets you avoid doing crazy things to make your empty-state variable map to desired defaults. Not super attached to it, but I like them